scripts/build-benchmark.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -eux
+
+set -e
+set -u
+set -x
+
+BENCHMARKS=(
+  # create-pages
+  # image-processing
+  markdown_id
+  markdown_slug
+  markdown_table
+  # plugin-manifest
+  # query
+)
+
+SIZES=(
+    1000
+    5000
+   10000
+   25000
+  100000
```

Comment:
This (100k) is going to run for a very long time. The 50k benchmark runs in ~50 minutes and the scaling hockeysticks. Not sure we need it right now, except for perhaps sites that can do this in under an hour.

Reply:
Commented (removed)